doc/nrf/releases/release-notes-1.4.0.rst

```diff
@@ -0,0 +1,1013 @@
+  * :ref:`bt_mesh_time_readme` - These models allow network-wide time and date synchronization.
+  * :ref:`bt_mesh_light_ctl_readme` - These models allow remote control and configuration of CTLs on a mesh device.
+  * :ref:`bt_mesh_scene_readme` - These models allow storing the model state of the entire mesh network as a *scene*, which can be recalled at a later time.
+
+* Added support for Mesh Device Properties v2.0.
+* Updated :ref:`bt_mesh_light_ctrl_readme`.
+  Light control regulator now uses floating point.
+* Fixed several bugs and improved documentation.
+
+
+nRF IEEE 802.15.4 radio driver
+------------------------------
+
+* Added support for multiprotocol with :ref:`nrfxlib:softdevice_controller`.
+
+Thread
+------
+
+* Added:
+
+  * Production support for nRF52833 and nRF52840 DKs.
+  * Dynamic multiprotocol support with Bluetooth LE.
+  * Support for :ref:`thread_architectures_designs_cp_ncp` architecture.
+  * Support for :ref:`ug_thread_vendor_hooks`.
+  * :ref:`ot_ncp_sample` sample - This sample demonstrates the usage of OpenThread Network Co-Processor architecture inside the Zephyr environment.
+    It has the following characteristics:
+
+    * Extendable with vendor hooks
+    * Hardware cryptography acceleration support
+    * Support for :ref:`Spinel logging <ug_logging_backends>` as a default logger backend
+    * UART hardware flow control (HWFC) enabled by default
+
+  * Initial support for :ref:`Thread 1.2 functionalities <thread_ug_supported_features_v12>` related to SED implementation and reduction of power consumption.
+
+* Enabled Thread v1.1 certification by inheritance.
+  For more information about certification, see :ref:`ug_thread_cert`.
+* Updated:
+
+  * :ref:`ot_cli_sample` sample:
+
+    * Switched to RNG peripheral as an entropy source.
+    * Added support for :ref:`Thread Certification Test harness <ug_thread_cert>`.
+    * Added hardware cryptography acceleration support.
+    * Enabled UART HWFC by default.
+    * Added the overlay for :ref:`Thread v1.2 support <ot_cli_sample_thread_v12>`.
+
+  * :ref:`coap_client_sample` sample:
+
+    * Switched to RNG peripheral as an entropy source.
+    * Optimized power in SED mode.
+    * Added hardware cryptography acceleration support.
+    * Added :ref:`multiprotocol support <coap_client_sample_multi_ext>`.
+    * Enabled UART HWFC by default.
+
+  * :ref:`coap_server_sample` sample:
+
+    * Switched to RNG peripheral as an entropy source.
+    * Added hardware cryptography acceleration support.
+    * Enabled UART HWFC by default.
+
+Zigbee
+------
+
+* Added:
+
+  * Production support for nRF52833 and nRF52840.
+  * :ref:`lib_zigbee_fota` (DFU) support for nRF52840.
+  * Support for Command Line Interface (:ref:`lib_zigbee_shell`).
+    This is a port from the nRF5 SDK for Thread and Zigbee.
+  * Support for :ref:`Network Co-Processor <ug_zigbee_platform_design_ncp>` architecture.
```

Comment:
This line probably needs to be removed (NCP cannot go public by the NCS release date).